### ISSUE TITLE
$stripe->products()->all();

### ISSUE THREAD

**Issue by KaminoU** [cartalyst/stripe#146]
### Your Environment

- OS and Version: Ubuntu 18.04.3
- PHP Version: 7.3.12
- Stripe Laravel Version: ^11.0

### Expected behaviour

an array of products

### Actual behaviour

empty array :
array:4 [▼
  "object" => "list"
  "data" => []
  "has_more" => false
  "url" => "/v1/products"
]

### Steps to reproduce

`dd($stripe->products()->all());`

Hi, 

I firstly though that I made some typo error but I am sure that I did well. Especially that when I focus on a specific product, it works great `$stripe->products()->find('product_string_id')`

I also tried with `$stripe->customers()->all();` and it also works like a charm...

Maybe I did some mistakes or misunderstanding =)

Thank you for any feedback and help

By the way, `php artisan vendor:publish --tag="stripe.billing.migrations"` raise a red alert **Unable to locate publishable resources**. and same for `php artisan stripe:schema` that returns **There are no commands defined in the "stripe" namespace.**

**Comment by KaminoU** [comment]
Thank you for your quick reply @brunogaspar 

I do not know what to say then :/
I believe that I can do without it but still... it's weird =)

I do not know if it would help or not, I flushed all my test data and even with new fresh one I still have the same behaviour (but only with products()->all())

products()->find() is OK and customers()->all() is also OK

**Comment by brunogaspar** [comment]
What is the API Version you're using?

Either the one from https://dashboard.stripe.com/developers or what you've might set on the config file.

I've tried with different versions, can't reproduce either and it has been working as expected for quite some time, since this products API was introduced on the library.

**Comment by KaminoU** [comment]
hum... as it's optional I must admit that I did not pay attention on it (it's the default one).

I will check it =)

And about the artisan command error, do you have any tips please?

**Comment by brunogaspar** [comment]
This package does not have any Artisan command.

**Comment by KaminoU** [comment]
You are right sorry, I read 2 differents page, was focused on my issue and didn't pay attention 

https://cartalyst.com/manual/stripe-billing-laravel/11.x#installation

And 

https://cartalyst.com/manual/stripe-laravel/11.x

**Comment by KaminoU** [comment]
Love U <3 @brunogaspar  =3

After forcing the version of the API, it works well.

I checked the log on Stripe console and set the 2019-11-05 as the default. Before, the default API version was 2018-09-06 and for a weird reason, without declaring the API version, the call of products()->all() trigger all the other API version (it's the info I got from the log). Maybe it explains why I have an empty array.

Anyway it works well now thank you for your time
